test/replication_connection_test.exs

```diff
@@ -107,11 +113,15 @@ defmodule ReplicationTest do
 
   describe "handle_result" do
     test "on result", context do
-      assert {:ok, %Postgrex.Result{}} = PR.call(context.repl, {:query, "SELECT 1"})
+      assert {:ok, [%Postgrex.Result{}]} = PR.call(context.repl, {:query, "SELECT 1"})
+    end
+
+    test "on multiple results", context do
+      assert {:error, %Postgrex.Error{}} = PR.call(context.repl, {:query, "SELECT 1; SELECT 2;"})
     end
```

Comment:
As I understand, this tests the postgres bug. Should we?

Reply:
Yeah, it is better not to test it as it will start breaking when we change CI?

Reply:
good call, removed